src/assets/fonts/acumin-pro/index.less

```diff
@@ -3,7 +3,7 @@
 //============================================================================
 
 .acumin-pro-defaults {
-   font-family: 'acumin-pro','Helvetica Neue','Helvetica','Arial','sans-serif';
+   //font-family: 'acumin-pro','Helvetica Neue','Helvetica','Arial','sans-serif';
```

Comment:
Removed. This whole file will likely be renamed and reformatted to not be specific to a particular font family.